### PR TITLE
Update Heading

### PR DESCRIPTION
**Pull request by wEprous**
This Pull Request meets the following criteria:

* [x] Tests have been added/adjusted for my new feature
* [x] New Components are registered in index.js of my project

### CODE REVIEW

**Comment by williamkolmacka** [comment]
Completed. You can check it.

File: src/Heading/index.js [hunk 1]
```diff
@@ -3,21 +3,22 @@ import * as React from "react";
...
+  title3: tokens.fontSizeHeadingTitle3,
 };
 
 type Props = {
   element: "h1" | "h2" | "h3" | "h4" | "h5",
-  size: $Keys<typeof sizeHeading>,
-  weight: $Keys<typeof weightHeading>,
+  type: $Keys<typeof weightHeading>,
   children: React.Node,
   className?: string,
```
Comment (tomashapl) [inline src/Heading/index.js]: Is `className` really necessary ? I thought we talked about wrapping component with classname and let component without classnames

Comment (williamkolmacka) [reply src/Heading/index.js]: Will fix this

File: src/Heading/index.js [hunk 2]
```diff
@@ -49,11 +52,11 @@ const Heading = (props: Props) => {
   return (
     <React.Fragment>
       {React.createElement(
-        props.element,
+        element,
         {
-          className: [scoped.className, props.className].join(" "),
+          className: [scoped.className].join(" "),
```
Comment (tomashapl) [inline src/Heading/index.js]: 🙏 remove join

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@kiwicom/orbit-components",
-  "version": "0.0.0-rc7",
+  "version": "0.0.0-rc2",
```
[the remark this answers is not in the document]
Comment (williamkolmacka) [reply package.json]: done

File: package.json [hunk 4]
```diff
@@ -59,12 +60,11 @@
     "@storybook/react": "^3.4.3",
     "@storybook/storybook-deployer": "^2.3.0",
     "babel-cli": "^6.26.0",
-    "babel-core": "^6.26.3",
```
Comment (tomashapl) [inline package.json]: Please use `master` version

File: package.json [hunk 5]
```diff
@@ -59,12 +60,11 @@
...
     "babel-eslint": "^8.2.3",
+    "babel-plugin-styled-components": "^1.5.1",
     "babel-plugin-transform-class-properties": "^6.24.1",
     "babel-preset-env": "^1.6.1",
     "babel-preset-react": "^6.24.1",
-    "babel-runtime": "^6.26.0",
```
Comment (tomashapl) [inline package.json]: Please use `master` version

**Pull request merged**